Documentation/En/Index.xyl

```diff
@@ -14,7 +14,7 @@
 
   <h2 id="Introduction" for="main-toc">Introduction</h2>
 
-  <p>Software quality is a vast topic in Computer Science. There is a lot of
+  <p>Software quality is a vast topic in Computer Science. There are a lot of
```

Comment:
No, that's more subtle. http://www.grammar.cl/Present/ThereIsThereAre.htm
“lot of strategies”, this is not countable, so “there is” is fine.